modules/33acrossBidAdapter.js

```diff
@@ -28,21 +30,28 @@ function _createBidResponse(response) {
 function _createServerRequest(bidRequest, gdprConsent) {
   const ttxRequest = {};
   const params = bidRequest.params;
+  const element = document.getElementById(bidRequest.adUnitCode);
+  const sizes = _transformSizes(bidRequest.sizes);
+  const minSize = _getMinSize(sizes);
+
+  const contributeViewability = ViewabilityContributor(
+    _getPercentInView(element, window.top, minSize)
+  );
 
   /*
    * Infer data for the request payload
    */
   ttxRequest.imp = [];
   ttxRequest.imp[0] = {
     banner: {
-      format: bidRequest.sizes.map(_getFormatSize)
+      format: sizes.map(size => Object.assign(size, {ext: {}}))
```

Comment:
Can you not use `_transformSizes` method here to construct the format sizes?

Reply:
`sizes` constant already has transformed sizes. So I just add `ext: {}` to each of them.